tests/model.test.js

```diff
@@ -138,7 +138,7 @@ test("EfficientNetModel - use different locale chinese", (done) => {
 
 test("EfficientNetModel - use different locale english", (done) => {
   EfficientNetCheckPointFactory.create(EfficientNetCheckPoint.B0, {
-    locale: EfficientNetLableLanguage.ENGLISH,
+    locale: EfficientNetLabelLanguage.ENGLISH,
   })
     .then(async (model) => {
```

Comment:
please add Hebrew test here

Reply:
added the test but for some reason, all tests of this kind on this page don't work (attaching error) and because it's out of the scope of the PR I won't be handling it at the time.
```

ReferenceError: fetch is not defined

    at HTTPRequest.Object.<anonymous>.PlatformBrowser.fetch (.....................\node-efficientnet\tfjs-core\src\platforms\platform_browser.ts:39:5)
    at HTTPRequest.<anonymous> (.....................\node-efficientnet\tfjs-core\src\io\http.ts:142:43)
    at step (.....................\node-efficientnet\node_modules\tslib\tslib.es6.js:102:23)
    at Object.next (.....................\node-efficientnet\node_modules\tslib\tslib.es6.js:83:53)
    at .....................\node-efficientnet\node_modules\tslib\tslib.es6.js:76:71
    at new Promise (<anonymous>)
    at __awaiter (.....................\node-efficientnet\node_modules\tslib\tslib.es6.js:72:12)
    at HTTPRequest.Object.<anonymous>.HTTPRequest.load (.....................\node-efficientnet\node_modules\@tensorflow\tfjs-core\dist\tf-core.node.js:8284:16)
    at GraphModel.Object.<anonymous>.GraphModel.load (.....................\node-efficientnet\tfjs-converter\src\executor\graph_model.ts:153:37)
    at Object.<anonymous> (.....................\node-efficientnet\tfjs-converter\src\executor\graph_model.ts:495:15)
    at step (.....................\node-efficientnet\node_modules\tslib\tslib.es6.js:102:23)
    at Object.next (.....................\node-efficientnet\node_modules\tslib\tslib.es6.js:83:53)
    at .....................\node-efficientnet\node_modules\tslib\tslib.es6.js:76:71
    at new Promise (<anonymous>)
    at __awaiter (.....................\node-efficientnet\node_modules\tslib\tslib.es6.js:72:12)
    at Object.loadGraphModel (.....................\node-efficientnet\node_modules\@tensorflow\tfjs-converter\dist\tf-converter.node.js:30681:12)
    at EfficientNetModel._callee$ (.....................\node-efficientnet\src\EfficientnetModel.ts:40:28)
    at tryCatch (.....................\node-efficientnet\src\EfficientnetModel.ts:26:2404)
    at Generator._invoke (.....................\node-efficientnet\src\EfficientnetModel.ts:26:1964)
    at Generator.next (.....................\node-efficientnet\src\EfficientnetModel.ts:26:3255)
    at asyncGeneratorStep (.....................\node-efficientnet\src\EfficientnetModel.ts:28:103)
    at _next (.....................\node-efficientnet\src\EfficientnetModel.ts:30:194)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Reply:
thanks a lot ! I fix this issue and meagre your PR